gen/gen.go

```diff
@@ -36,7 +36,7 @@ func New() *Gen {
 
 // Config presents Gen configurations.
 type Config struct {
-	// SearchDir the swag would be parse
+	// SearchDir the swag would be parse,comma separated if multiple
```

Comment:
If we want to support search muli directories. I think adding a new one flag is better than using the original flag.

Reply:
I need this feature - is it implemented somewhere (in any other developer branch)?